tcf_website/urls.py

```diff
@@ -20,6 +21,10 @@
     path('instructor/<int:instructor_id>',
          views.instructor_view, name='instructor'),
     path('reviews/new', views.new_review, name='new_review'),
+    path(
+        'reviews/new/subdept/<int:subdepartment_id>/course/<int:course_id>/instr/<int:instructor_id>',
```

Comment:
Since you're already using the names `subdept`, `instr` in the path, what do you think of modifying the URL parameter names like`<int:subdepartment_id>` to be consistent? Maybe the abbreviation will shorten the line so Pylint no longer complains.

On the other hand, it does kinda break with the naming conventions established by the actual models e.g. `Course.subdepartment` so maybe not.